connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties

```diff
@@ -177,7 +179,9 @@ datasources.data-source.enlistment-trace=Defines if WildFly/IronJacamar should r
 datasources.data-source.recovery-username=The user name used for recovery
 datasources.data-source.recovery-password=The password used for recovery
 datasources.data-source.recovery-security-domain=The security domain used for recovery, for PicketBox security domains
+datasources.data-source.recovery-security-domain.deprecated=PicketBox security domains are not supported in servers running the current version of the datasources subsystem. Use 'authentication-context' to configure Elytron security
 datasources.data-source.recovery-elytron-enabled=Enables Elytron security for handling authentication of connections for recovery. The Elytron authentication-context to be used will be current context if no context is specified (see authentication-context).
+datasources.data-source.recovery-elytron-enabled.deprecated=Elytron is enabled by default. Setting this flag to false will cause validation to fail.
```

Comment:
@tadamski Why is the semantic for this different? Shouldn't it just be that in a current version server recovery-elytron-enabled is ignored and setting recovery-security-domain fails?

XaDataSourceAdd.performRuntime looks like it's still trying to reason based on the value of recovery-elytron-enabled.

Reply:
@bstansberry hmm, actually this is a docs error; my intention was it to work just as you described

Reply:
fixed